syncstorage/storage/sql/dbconnect.py

```diff
@@ -102,6 +102,22 @@ def __new__(cls, *args, **kwds):
 )
 
 
+# Table listing migration state for users
+#
+# This table contains the users in progress for migration. This table may
+# be updated by the external user migration script
+# A UID appearing in this table should cause the server to return a 5xx
+# response. It's important that the server NEVER return a 2xx response.
+
+migration = Table(
+    "migration",
+    metadata,
+    Column("fxa_uid", String(255), primary_key=True, nullable=False),
+    Column("started_at", BigInteger, nullable=False),
+    Column("state", String(32))  # unknown/NULL, in_progress, complete
```

Comment:
@Micheletto, any concern about the space overhead of using a string here rather than e.g. a smallint enum?

Reply:
FWIW, I'm fine with whatever. I just need to make sure that this is coordinated with the other PRs related to migration.

Reply:
Bumping this

Reply:
@jrconlin do we need to open a new PR here(and issue) to get this change in since it looks like this merged already?

Reply:
Yes. see https://github.com/mozilla-services/syncstorage-rs/issues/428 &  https://github.com/mozilla-services/server-syncstorage/issues/143